### PR TITLE
Create create_private_thread_MOD.js

### PR DESCRIPTION
**Pull request by Phyrok**
**Please describe the changes this PR makes and why it should be merged:**
New Mod Private Thread

**Status**

- [x] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [x] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by TheMonDon** [comment]
DBM 2.2.0 already has private threads there's no reason to create a mod

**Comment by DetroitWhiskey136** [comment]
you could probably move all of this into a single action... using tabs or something.

**Review by Phyrok — COMMENTED** [review]
Updated ESLint

**Review by TheMonDon — APPROVED** [review]

File: actions/remove_member_from_thread_MOD.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  name: 'Remove Member to Thread',
```
Comment (TheMonDon) [inline actions/remove_member_from_thread_MOD.js:2]:
```suggestion
  name: 'Remove Member from Thread',
```

File: docs/mods.json [hunk 2]
```diff
@@ -958,5 +958,20 @@
...
+    "name": "Create Private Thread",
+    "section": "Channel Control"
+  },
+  "add_member_to_thread_MOD.js": {
+    "description": "Add Member To Private Thread",
+    "name": "Add Member To Private Thread",
+    "section": "Channel Control"
+  },
+  "remove_member_from_thread_MOD.js": {
+    "description": "Remove Member From Private Thread",
```
Comment (TheMonDon) [inline docs/mods.json:973]:
```suggestion
    "description": "Remove Member From Thread",
```

File: docs/mods.json [hunk 3]
```diff
@@ -958,5 +958,20 @@
...
     "name": "Create Forum Channel",
     "section": "Channel Control"
+  },
+  "ccreate_private_thread_MOD.js": {
+    "description": "Create Private Thread",
+    "name": "Create Private Thread",
+    "section": "Channel Control"
+  },
+  "add_member_to_thread_MOD.js": {
+    "description": "Add Member To Private Thread",
```
Comment (TheMonDon) [inline docs/mods.json:968]:
```suggestion
    "description": "Add Member To Thread",
```

**Review by TheMonDon — CHANGES_REQUESTED** [review]

**Review by OneAndonlyFinbar — APPROVED** [review]

**Pull request merged**